src/org/traccar/helper/LogAction.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017 Anton Tananaev (anton@traccar.org)
+ * Copyright 2017 Andrey Kunitsyn (andrey@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.helper;
+
+import org.traccar.model.BaseModel;
+
+public final class LogAction {
+
+    private LogAction() {
+    }
+
+    private static final String ACTION_CREATE = "create";
+    private static final String ACTION_EDIT = "edit";
+    private static final String ACTION_REMOVE = "remove";
+
+    private static final String ACTION_LINK = "link";
+    private static final String ACTION_UNLINK = "unlink";
+
+    private static final String ACTION_LOGIN = "login";
+    private static final String ACTION_LOGOUT = "logout";
+
+    private static final String ACTION_TOTAL_DISTANCE = "resetTotalDistance";
+
+    private static final String PATTERN_OBJECT = "User: %d, Action: %s, Object: %s, Id: %d";
```

Comment:
I think it would be better to use low-case words everywhere. It will make it consistent with location info logging.

Reply:
Should I also "toLowerCase" class names or it is OK?

```
2017-12-19 12:13:33  INFO: user: 1, action: login
2017-12-19 12:13:45  INFO: user: 1, action: edit, object: Device, id: 36
2017-12-19 12:13:59  INFO: user: 1, action: edit, object: Server, id: 1
2017-12-19 12:14:32  INFO: user: 1, action: link, owner: Device, id: 36, property: Geofence, id: 5
```

Reply:
I think classname are fine. Or maybe we can make them lower-camel-case for consistency.